### PR TITLE
Add testutils package

### PR DESCRIPTION
**Pull request by GinnyJI**
Refactor e2e test related helper functions into pkg/testutils.go

Tested: make e2e-test

### CODE REVIEW

File: incubator/hnc/pkg/testutils.go [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+)
+
+// The time that Eventually() will keep retrying until timeout
+// we use 5 seconds here because some tests require deleting a namespace, and shorter time period might not be enough
+const eventuallyTimeout = 5
+
+var HncRecoverPath = os.Getenv("HNC_REPAIR")
```
[the remark this answers is not in the document]
Comment (GinnyJI) [reply incubator/hnc/pkg/testutils.go:18]: This is actually used outside the package, in issue_test.go line 277.
If we want to keep this variable unexported here, we can have the line #277 call RecoverHNC() instead

[the remark this answers is not in the document]
Comment (GinnyJI) [reply incubator/hnc/pkg/testutils.go]: I think change it into an unexported variable and update `issue_test.go` to call `RecoverHNC()` looks like a better idea. Changed the code

File: incubator/hnc/pkg/testutils.go [hunk 2]
```diff
@@ -0,0 +1,136 @@
...
+	}, eventuallyTimeout).Should(Not(BeNil()))
+}
+
+func TryRun(cmdln ...string) error {
+	stdout, err := RunCommand(cmdln...)
+	GinkgoT().Log("Output: ", stdout)
+	return err
+}
+
+func TryRunSuppressLog(cmdln ...string) error {
```
[the remark this answers is not in the document]
Comment (GinnyJI) [reply incubator/hnc/pkg/testutils.go]: done

File: incubator/hnc/pkg/testutils.go (hunk 1 above)
[the remark this answers is not in the document]
Comment (adrianludwin) [reply incubator/hnc/pkg/testutils.go]: lgtm. For future reference, if a Go name starts with an acronym, the entire acronym should have the same case - e.g., `hncRecoverPath` or `HNCRecoverPath`, but not `HncRecoverPath`.

**Pull request merged**